src/hera/task.py

```diff
@@ -341,7 +342,14 @@ def get_parameters(self) -> List[V1alpha1Parameter]:
             args = set(inspect.getfullargspec(self.func).args).intersection(set(self.input_from.parameters))
             for arg in args:
                 parameters.append(V1alpha1Parameter(name=arg, value=f'{{{{item.{arg}}}}}'))
+        if self.func:
+            parameters += self.get_func_parameters()
+
+        return parameters
 
+    def get_func_parameters(self) -> List[V1alpha1Parameter]:
```

Comment:
Last thing we need is docstrings for `get_func_parameters` and then we can ship! :shipit: